tests/foundry/exclude

```diff
@@ -168,3 +159,35 @@ UintTypeTest.test_uint80_fail
 UintTypeTest.test_uint88_fail
 UintTypeTest.test_uint8_fail
 UintTypeTest.test_uint96_fail
+UintTypeTest.test_uint256
+UintTypeTest.test_uint248
+UintTypeTest.test_uint240
+UintTypeTest.test_uint232
+UintTypeTest.test_uint224
+UintTypeTest.test_uint216
+UintTypeTest.test_uint208
+UintTypeTest.test_uint200
+UintTypeTest.test_uint192
+UintTypeTest.test_uint184
+UintTypeTest.test_uint176
+UintTypeTest.test_uint168
+UintTypeTest.test_uint160
+UintTypeTest.test_uint152
+UintTypeTest.test_uint144
+UintTypeTest.test_uint136
+UintTypeTest.test_uint128
+UintTypeTest.test_uint120
+UintTypeTest.test_uint112
+UintTypeTest.test_uint104
+UintTypeTest.test_uint96
+UintTypeTest.test_uint88
```

Comment:
We can probably keep _one_ of these tests? You maybe already have kept one of them?